docs/sources/operators-guide/deploy-grafana-mimir/getting-started-helm-charts/_index.md

```diff
@@ -79,6 +79,11 @@ Using a custom namespace solves problems later on because you do not have to ove
          # empty, disabled.
    ```
 
+   > **Note:** You can use `helm show values grafana/mimir-distributed` to see all configurable parameters for helm install.
+   > However, refrain from copy and paste the whole thing because all parameters default value are already set and you should only
+   > override the parameter that you want to change. Likewise, `values.yaml` from `operations/helm/charts/mimir-distributed/values.yaml`
```

Comment:
Let's be more explicit: if you want to get the values file from github, you should build the URL like this:
`https://github.com/grafana/mimir/blob/mimir-distributed-<chart-version>/operations/helm/charts/mimir-distributed/values.yaml`
e.g.: https://github.com/grafana/mimir/blob/mimir-distributed-3.1.0/operations/helm/charts/mimir-distributed/values.yaml

Because we git tag every release.

Reply:
For now I put it the direct link to main branch. Or should I put link to particular version?